Services/ApplicationLogService.php

```diff
@@ -7,14 +7,29 @@
 
 class ApplicationLogService extends LogService
 {
+    /**
+     * @var string
+     */
+    private $streamName;
+
+    /**
+     * ApplicationLogService constructor.
+     * @param string $applicationName
+     * @param string $environment
+     */
+    public function __construct(string $applicationName, string $environment)
+    {
+        $this->streamName = $applicationName . '-' . $environment . ' -application';
+    }
+
     /**
      * @return array
      */
     protected function loggersMap(): array
     {
         return [
-            self::LOG_TO_FILE => function () {return (new FileLoggingRegistry())->logger(LOG_PATH . 'application.log', 'application');},
-            self::LOG_TO_CLOUD_WATCH => function () {return (new CloudWatchLoggingRegistry())->logger('php-logs', 'application', 'core');},
+            self::LOG_TO_FILE => function () {return (new FileLoggingRegistry())->logger(LOG_PATH . $this->streamName . '.log', 'application');},
+            self::LOG_TO_CLOUD_WATCH => function () {return (new CloudWatchLoggingRegistry())->logger('php-logs', $this->streamName, 'core');},
```

Comment:
can you do this configurable ? 
this is hardcode

Reply:
it  makes additional unjustified complexity at this stage, we can do it when to appear a necessity